app/src/main/java/org/mozilla/fenix/ext/Fragment.kt

```diff
@@ -39,6 +40,7 @@ fun Fragment.getPreferenceKey(@StringRes resourceId: Int): String = getString(re
  * Throws if the fragment is not attached to an [AppCompatActivity].
  */
 fun Fragment.showToolbar(title: String) {
+    (activity as? HomeActivity)?.getSupportActionBarAndInflateIfNecessary()
```

Comment:
Why not replace the `supportActionBar` in `supportActionBar.show()` line with this?